pkg/cmd/browse/browse_test.go

```diff
@@ -45,37 +136,87 @@ func Test_runBrowse(t *testing.T) {
 			name: "no arguments",
 			opts: BrowseOptions{
 				SelectorArg: "",
-				FlagAmount:  0,
 			},
-			baseRepo:    ghrepo.New("jessica", "cli"),
-			expectedURL: "https://github.com/jessica/cli",
+			baseRepo:      ghrepo.New("jessica", "cli"),
+			defaultBranch: "trunk",
+			expectedURL:   "https://github.com/jessica/cli/tree/trunk/",
```

Comment:
It looks like the no-arguments scenario now browses `REPO/tree/BRANCH` instead of simply the repository homepage. Could we go back to opening the repository homepage for no-arguments scenario? The homepage will show the default branch anyway, so we save ourselves having to do an API call.